vue-library/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "./dist",
     "lib": ["dom", "es2020"],
-    "module": "es2015",
+    "module": "commonjs",
```

Comment:
We might want to come back to this later and try to get this to work as es2015 again as it is old but still more modern than common js

Reply:
actually, I figured it out this change is not necessary. I am reverting it